stash-cache/README.md

```diff
@@ -33,18 +64,3 @@ the following in your environment variable configuration:
 DISABLE_OSG_MONITORING = true
 ```
 
-Running a Container
--------------------
-
-To run the container, use `docker run` with the following options, replacing the text within angle brackets with your
-own values:
-
-
-```
-$ docker run --env-file=<PATH TO ENV FILE> \
-             --volume <PATH TO HOST CERT>:/etc/grid-security/hostcert.pem \
-             --volume <PATH TO HOST KEY>:/etc/grid-security/hostkey.pem \
-             --publish <HOST PORT>:<XC_PORT> \
-             --hostname <XCACHE HOSTNAME> \
-             opensciencegrid/stash-cache:development
-```
```

Comment:
I'd prefer that we don't completely get rid of this section since it's necessary for running a production cache